BUILD.gn

```diff
@@ -20,6 +20,12 @@ config("deno_config") {
   if (is_debug) {
     defines = [ "DEBUG" ]
   }
+  if (is_mac) {
+    libs = [ "resolv" ]
+  }
+  if (is_win) {
+    libs = [ "userenv.lib" ]
+  }
```

Comment:
Are these not discovered with the new script?

Reply:
They are, but test_cc still needs it here, because it's not linked with rust_executable.

Reply:
Maybe add a comment to that effect

Reply:
done